### ISSUE TITLE
Add an option to always see the raw XML

### ISSUE THREAD

**Issue by personalmountains** [aureliendavid/rsspreview#29]
The main reason I open RSS feeds in Firefox is to see their markup for debugging. I've never needed previews in the browser. Unfortunately, Firefox doesn't even interpret an RSS feed as a text file now, it just downloads it.

Your extension is really useful for bringing back the original behaviour, but I would love to have an option to always see the feed's source without having to load the preview and then press CTRL+U. Some feeds are also on one single line, so adding a prettifier would be even better ;)

**Comment by aureliendavid** [comment]
Hi, 

It's a bit counter-intuitive to add an option not to do what the extension is meant for :)

but I understand the problem of firefox downloading feeds (which is annoying!) and it doesn't cost much to do so I added it

if the option is checked, the document should be displayed with the firefox xml viewer so it should be prettified 

available from v3.4

**Comment by personalmountains** [comment]
That's fine. I was hoping to see the source directly, but it probably makes more sense to keep the standard behaviour of CTRL+U for source. I tried setting [`layout.xml.prettyprint`](http://kb.mozillazine.org/Layout.xml.prettyprint) to `false`, but this actually shows the XML file as HTML instead of the source, which is useless.

Since Firefox's viewer is kinda crap, I searched for a replacement and found [Pretty XML](https://addons.mozilla.org/en-US/firefox/addon/pretty-xml/) ([on github](https://github.com/Sytten/PrettyXML)). It seems to work fine.

Thank you for implementing my two requests so quickly!
